Android/Guava/source/Guava.FailureAccess/Guava.FailureAccess.csproj

```diff
@@ -20,38 +20,30 @@
     <Title>Google Guava FailureAccess reference library for Xamarin.Android</Title>
     <Summary>Xamarin.Android bindings for Google Guava FailureAccess</Summary>
     <Description>
-    NOTE: This package is meant to be used as a reference only for other binding projects that depend on Guava.
-    This package does not expose any useful API's in the .dll itself.
-    
-    Guava is a set of core libraries that includes new collection types (such as multimap and multiset), immutable collections, a graph library, functional types, an in-memory cache, and APIs/utilities for concurrency, I/O, hashing, primitives, reflection, string processing, and much more!
+NOTE: This package is meant to be used as a reference only for other binding projects that depend on Guava.
+This package does not expose any useful API's in the .dll itself.
+
+Guava is a set of core libraries that includes new collection types (such as multimap and multiset), immutable collections, a graph library, functional types, an in-memory cache, and APIs/utilities for concurrency, I/O, hashing, primitives, reflection, string processing, and much more!
     </Description>
     <Authors>Microsoft</Authors>
     <Owners>Microsoft</Owners>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=865028</PackageProjectUrl>
     <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=865030</PackageLicenseUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <PackageVersion>1.0.1</PackageVersion>
+    <PackageVersion>1.0.1.1</PackageVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <TransformFile Include="Transforms\Metadata.xml" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Include="..\..\externals\guava-failureaccess.jar">
-      <Link>guava-failureaccess.jar</Link>
-    </None>
+    <None Include="Guava.FailureAccess.targets" Pack="True" PackagePath="build\$(TargetFramework)\$(PackageId).targets" />
+    <None Include="..\..\externals\guava-failureaccess.jar" Pack="True" PackagePath="build\$(TargetFramework)\guava-failureaccess.jar" />
     <None Include="..\..\External-Dependency-Info.txt" Pack="True" PackagePath="THIRD-PARTY-NOTICES.txt" />
   </ItemGroup>
 
-  <ItemGroup>
-    <EmbeddedJar Include="..\..\externals\guava-failureaccess.jar" />
```

Comment:
We can't embed the jar as Release builds make sure to remove all assemblies that are not referenced, and this assembly will never be referenced - it can't be as there are no types to reference.
Also, no need to extract the jar, that is just extra work.